user_management/views/menuitem.xml

```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
     <data>
+        <!-- Admin needs to be able to see this menu.
+             Adding the dev group was the only was to make this happen.
```

Comment:
Tiny thing from code review: 'was the only way' not 'was the only was' Otherwise, good to go :shipit: 
Might be worth getting a group for Admin that is not 'dev'? Unsure if you'd want to raise this as tech debt and look at it later @moderatemisbehaviour